packages/assets-controllers/src/token-service.ts

```diff
@@ -78,7 +82,10 @@ export async function fetchTokenMetadata<T>(
   abortSignal: AbortSignal,
   { timeout = defaultTimeout } = {},
 ): Promise<T | undefined> {
-  if (!isTokenListSupportedForNetwork(chainId)) {
+  if (
+    !isTokenListSupportedForNetwork(chainId) ||
```

Comment:
Sorry, I don't remember what was here before because you squashed the commit. If the goal of this function is to determine whether a chain supports token lists, what do you think about modifying the `isTokenListSupportedForNetwork` function to exclude the `GANACHE_CHAIN_ID`? Or did it cause the e2e test to fail as you mentioned? If that's the case, is there something we can do on the extension side to fix the test rather than making this code less cohesive?

Reply:
I have made an update so the fct `isTokenListSupportedForNetwork` does not consider the GanacheID as supported; (since the call inside fetchTokenMetadata to tokenApi would fail because token api does not support it anyway)

but also so that does not stop the test from doing the startPolling..